### PR TITLE
[ETP-548] Update display Setting's help text for Tickets

### PR DESCRIPTION
**Pull request by rafsuntaskin**
[ETP-548]

Screen Shot 2020-11-02 at 11 34 43 AM

[ETP-548]: https://moderntribe.atlassian.net/browse/ETP-548

### CODE REVIEW

File: src/admin-views/tribe-options-display.php [hunk 1]
```diff
@@ -134,10 +134,10 @@
...
 			)
 		),
-		'tooltip'         => esc_html(
+		'tooltip'         => wp_kses_post(
 			sprintf(
 			// Translators: %1$s: dynamic "Tickets" text.
-				_x( 'This setting will enable the updated front-end views and Individual Attendee Collection (IAC) flows for %1$s.', 'settings tooltip', 'event-tickets' ),
+				_x( 'Enables updated front-end views and Individual Attendee Collection (IAC) flows for %1$s. You\'ll see a new default setting within the Tickets tab and new individual ticket setting within both Classic and Gutenberg editors <a href="https://m.tri.be/kb">[Learn more]</a>.', 'settings tooltip', 'event-tickets' ),
 				tribe_get_ticket_label_plural( 'tickets_use_new_views' )
 			)
```
Comment (paulmskim) [inline src/admin-views/tribe-options-display.php:142]: move the anchor tags out into the sprintf so they are not required to be translated, and also allows moving around of the anchor tags in languages where the grammar doesn't work the same as English, something like:

```
sprintf(
// Translators: %1$s: dynamic "Tickets" text.
	_x( 'Enables updated front-end views and Individual Attendee Collection (IAC) flows for %1$s. You\'ll see a new default setting within the Tickets tab and new individual ticket setting within both Classic and Gutenberg editors %2$s[Learn more]%3$s.', 'settings tooltip', 'event-tickets' ),
	tribe_get_ticket_label_plural( 'tickets_use_new_views' ),
	<a href="https://m.tri.be/kb">,
	</a>
)
```

**Review by paulmskim — CHANGES_REQUESTED** [review]

File: src/admin-views/tribe-options-display.php [hunk 2]
```diff
@@ -134,11 +134,13 @@
...
-		'tooltip'         => esc_html(
+		'tooltip'         => wp_kses_post(
 			sprintf(
-			// Translators: %1$s: dynamic "Tickets" text.
-				_x( 'This setting will enable the updated front-end views and Individual Attendee Collection (IAC) flows for %1$s.', 'settings tooltip', 'event-tickets' ),
-				tribe_get_ticket_label_plural( 'tickets_use_new_views' )
+				// Translators: %1$s: dynamic "Tickets" text, %2$s and %3$s are part of link.
+				_x( 'Enables updated front-end views and Individual Attendee Collection (IAC) flows for %1$s. You\'ll see a new default setting within the Tickets tab and new individual ticket setting within both Classic and Gutenberg editors %2$s[Learn more]%3$s.', 'settings tooltip', 'event-tickets' ),
+				tribe_get_ticket_label_plural( 'tickets_use_new_views' ),
+				'<a href="https://m.tri.be/1an-">',
```
Comment (paulmskim) [inline src/admin-views/tribe-options-display.php:142]: This link changed when making the `sprintf()` change, is that correct?

**Review by paulmskim — APPROVED** [review]
Just one question, otherwise looks good.

File: src/admin-views/tribe-options-display.php (hunk 2 above)
Comment (sc0ttkclark) [reply src/admin-views/tribe-options-display.php:142]: Yes, we just got the real link ready

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**